### PR TITLE
restructed serial communication for increased robustness

### PR DESCRIPTION
**Pull request by HPovoa**
Some of these changes should fix concurrency issues and make the code a lot more stable.

### CODE REVIEW

**Comment by ipa-nhg** [comment]
please solve the merge conflict:
```
roscd squirrel_interaction
git pull https://github.com/squirrel-project/squirrel_driver
git mergetool
```

**Comment by ipa-nhg** [comment]
@bajo @HPovoa please merge this with the robot version together, test on the robot and update this pull request

File: squirrel_interaction/src/squirrel_interaction/board/serial_api.py [hunk 1]
```diff
@@ -71,16 +84,41 @@ def move_to(self, motor, degrees):
...
 
+    def start_motors(self):
+        """ Sets initial speeds for all Motors """
+	self.reset()
+        for motor in _MOTORS:
+            self.start_motor(motor)
+
+    def stop_motor(self, motor):
+        """ Stops motor [param 0] """
+        try:
```
Comment (bajo) [inline squirrel_interaction/src/squirrel_interaction/board/serial_api.py:94]: @HPovoa Should we acquire the lock here and in start_motors as well. Just to be safe.

**Comment by bajo** [comment]
@ipa-nhg Tested on the robot. Works fine. Ready to merge from our side.

File: squirrel_interaction/CMakeLists.txt [hunk 2]
```diff
@@ -12,21 +12,25 @@ find_package(catkin REQUIRED COMPONENTS
...
+  std_msgs
 )
 
 catkin_package(
   CATKIN_DEPENDS message_runtime
 )
 
 catkin_install_python(PROGRAMS 
-    src/squirrel_interaction/board/board.py src/squirrel_interaction/board/teleop.py src/squirrel_interaction/screen/raspberry_screen_server.py src/squirrel_interaction/speakers/play_sound.py src/squirrel_interaction/speakers/playsound.py
+    src/squirrel_interaction/board/board.py src/squirrel_interaction/teleop.py src/squirrel_interaction/screen/raspberry_screen_server.py src/squirrel_interaction/speakers/play_sound.py
```
Comment (NadiaHG) [inline squirrel_interaction/CMakeLists.txt]: mouth_speak.py script is missed

Comment (HPovoa) [reply squirrel_interaction/CMakeLists.txt]: @ipa-nhg it's added now

**Review by ipa-nhg — APPROVED** [review]

**Pull request merged**